src/test/java/HelperClass/ParserTest.java

```diff
@@ -0,0 +1,19 @@
+package HelperClass;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+public class ParserTest {
+    @Test
+    public void testGetUserCommand() {
+        Parser parser = new Parser();
+        parser.processUserCommand("todo Read Book");
+        assertEquals("todo", parser.getCommand());
+    }
+
+    @Test
+    public void testGetTaskName() {
+        Parser parser = new Parser();
+        parser.processUserCommand("todo Read Book");
+        assertEquals("Read Book", parser.getTaskName());
+    }
```

Comment:
According to [CS2103T Website W3.7d](https://nus-cs2103-ay2324s1.github.io/website/schedule/week3/topics.html#w3-7-unit-testing) code normally use camelCase for method names e.g., `testStringConversion` but when writing test methods, sometimes another convention is used:`unitBeingTested_descriptionOfTestInputs_expectedOutcome`.